v2/argsparser.go

```diff
@@ -117,6 +130,9 @@ parseLoop:
 			case r == '0':
 				buf.WriteRune(r)
 				st = stPrefixedNumber
+			case r == '-':
+				buf.WriteRune(r)
+				st = stNumber
```

Comment:
@thockin Should we allow negative numbers?  I've added it here.

Reply:
Do we have a use-case?  If we negative, do we also need '+' ?

Also I would write this as:

```
case r == '-' || r == '+' || unicode.IsDigit(r):
    buf.WriteRune(r)
    st = stNumber
```

...just to save on space

Reply:
Don't need negatives today, but since we are handling hex and octal, I threw it in for completeness.

Added + support.